src/Components/Videocard/Videocard.jsx

```diff
@@ -103,58 +107,117 @@ export const Videocard = ({ ele }) => {
               </li>
             </ul>
           ) : (
-            <ul className="flex-column gap-0-5">
-              <li>
-                {user_data.liked_video.find((item) => item?._id === ele._id) ? (
-                  <div className="flex mini-video-option gap-0-5">
-                    <i
-                      onClick={() => {
-                        removeLikeVideo(ele._id, handleaddtoast, setUser_Data);
-                        setShowOptions((pre_state) => !pre_state);
-                      }}
-                      className="fas fa-thumbs-up cursor-pointer"
-                    ></i>
-                    <p>Remove from Liked Video</p>
-                  </div>
-                ) : (
-                  <div className="flex mini-video-option gap-0-5">
-                    <i
-                      onClick={likeHandler}
-                      className="far fa-thumbs-up cursor-pointer"
-                    ></i>
-                    <p>Add to Like Video</p>
-                  </div>
-                )}
-              </li>
-              <li>
-                {user_data.watch_later.find((item) => item?._id === ele._id) ? (
-                  <div className="flex mini-video-option gap-0-5">
-                    <i
-                      onClick={() => {
-                        removeWatchLater(ele._id, handleaddtoast, setUser_Data);
-                        setShowOptions((pre_state) => !pre_state);
-                      }}
-                      className="fas fa-heart cursor-pointer"
-                    ></i>
-                    <p>Remove from Watch Later</p>
-                  </div>
-                ) : (
-                  <div className="flex mini-video-option gap-0-5">
-                    <i
-                      onClick={watchLaterHandler}
-                      className="far fa-heart cursor-pointer cursor-pointer"
-                    ></i>
-                    <p>Add to Watch Later</p>
-                  </div>
-                )}
-              </li>
-              <li>
-                <div className="flex mini-video-option gap-0-5">
-                  <i className="fas fa-plus cursor-pointer"></i>
-                  <p>Add to Playlist</p>
-                </div>
-              </li>
-            </ul>
+            <>
+              {location.substring(0, 5) === "/play" ? (
+                <ul className="flex-column gap-0-5">
+                  <li>
+                    <div className="flex mini-video-option gap-0-5">
+                      <i
+                        onClick={async () => {
```

Comment:
suggestion: you can also do explicit function here

Reply:
Done ..